DMX.Gatekeeper.Api/Models/LabWorkflows/Exceptions/AlreadyExistsLabWorkflowException.cs

```diff
@@ -0,0 +1,24 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// ---------------------------------------------------------------
+
+using System;
+using System.Collections;
+using Xeptions;
+
+namespace DMX.Gatekeeper.Api.Models.LabWorkflows.Exceptions
+{
+    public class AlreadyExistsLabWorkflowException : Xeption
+    {
+        public AlreadyExistsLabWorkflowException(Exception exception)
+            : base(message: "Lab workflow already exists. Please fix and try again.",
```

Comment:
You don't need to say `Please fix and try again` here. Just a simple message: `Lab workflow already exists.` just like we say `Lab workflow is null.`

Reply:
Canadian politeness is leaking into the code lol